content/geoip/docs/web-services/sandbox-environment.mdx

```diff
@@ -0,0 +1,59 @@
+---
+draft: false
+title: Testing GeoIP Web Services in the MaxMind Sandbox
+---
+
+import SandboxIntro from '/content/_sandbox-intro.mdx';
+import SandboxLicenseKey from '/content/_sandbox-license-key.mdx';
+import SandboxLimits from '/content/_sandbox-limits.mdx';
+import SandboxLearnMore from '/content/_sandbox-learn-more.mdx';
+
```

Comment:
What would lines 6 to 9 look like in a doc?

Reply:
lines 6-9 import the content listed in those lines when the call is made in the document

so, for example, line 11 is:

`<SandboxIntro />`

this references line 6, where we have imported `/content/sandbox-intro.mdx`.

so line 11 in this file would be replaced with the full contents of `/content/sandbox-intro.mdx`.

you can see how this looks in practice with the preview of the site:

https://mm-dev-site-staging--pr-727-bjx2ccwa.web.app/geoip/docs/web-services/sandbox-environment?lang=en